src/components/TaskHeadPanel.js

```diff
@@ -79,12 +79,14 @@ class TaskHeadPanel extends FormPanel {
             />
           </Grid>
           <Grid item xs={3} className={classes.item}>
-            <TextInput
+            <TaskGroupPicker
               module="tasksManagement"
-              label="benefitPlanTask.assignee"
-              readOnly={readOnly}
-              value={task?.assignee}
-              onChange={(assignee) => this.updateAttribute('assignee', assignee)}
+              mandatory
```

Comment:
We should use _required_ here instead of _mandatory_. Correct me if I'm wrong.

Reply:
you are right, done